src/Identity/test/Identity.FunctionalTests/Microsoft.AspNetCore.Identity.FunctionalTests.csproj

```
@@ -34,19 +37,23 @@
     </ItemGroup>
     <Copy
       SourceFiles="@(_PublishFiles)"
-      DestinationFolder="$(PublishDir)" />
+      DestinationFolder="$(PublishDir)"
+      ContinueOnError="true" />
```

Same question about failing fast.